### PR TITLE
feat(docs): serve for Firebase Cloud Functions

### PR DESCRIPTION
**Pull request by charlypoly**
Follow-up on https://discord.com/channels/842170679536517141/1257982036950515774

### CODE REVIEW

File: pages/docs/sdk/serve.mdx [hunk 1]
```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
...
+  serve({
+    client: inngestClient,
+    functions: [/* ...functions... */],
+  })
+);
+```
+
+<Callout>
+
+Firebase Cloud Function requires to configure the `INNGEST_SERVE_PATH` with the custom function path.
```
Comment (sylwiavargas) [inline pages/docs/sdk/serve.mdx:420]:
```suggestion
Firebase Cloud Functions require configuring `INNGEST_SERVE_PATH` with the custom function path.
```

"Require" is a transitive verb that needs a direct object to complete its meaning when followed by an infinitive phrase. By changing the phrase to gerund, we are providing a direct object.

File: pages/docs/sdk/serve.mdx [hunk 2]
```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
...
+
+For example, for a project named `inngest-firebase-functions` deployed on the `us-central1` region, the `INNGEST_SERVE_PATH` value will be as follows:
+
+```
+/inngest-firebase-functions/us-central1/inngest/
+```
+
+</Callout>
+
+You can now server your Firebase Cloud Function locally using the following command:
```
Comment (sylwiavargas) [inline pages/docs/sdk/serve.mdx:430]:
```suggestion
To serve your Firebase Cloud Function locally, use the following command:
```

"You can now" sounds conversational and given that this is not a tutorial but a reference page, we provide directions.

File: pages/docs/sdk/serve.mdx [hunk 3]
```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
...
+
+```bash
+firebase emulators:start
+```
+
+Please note that you'll need to start your Inngest Local Dev Server with the `-u` flag as follows:
```
Comment (sylwiavargas) [inline pages/docs/sdk/serve.mdx:436]: Could you add a reason for that?

Comment (charlypoly) [inline pages/docs/sdk/serve.mdx:436]:
```suggestion
Please note that you'll need to start your Inngest Local Dev Server with the `-u` flag as follows to match our Firebase Cloud Function's custom path:
```

Comment (charlypoly) [reply pages/docs/sdk/serve.mdx]: Updated!

**Comment by charlypoly** [comment]
@sylwiavargas, thank you for the review! 🦅

File: pages/docs/sdk/serve.mdx [hunk 4]
```diff
@@ -397,6 +397,50 @@ ff.http('inngest', serve(inngest, [fnA]));
...
+firebase emulators:start
+```
+
+Please note that you'll need to start your Inngest Local Dev Server with the `-u` flag to match our Firebase Cloud Function's custom path  as follows:
+
+```bash
+npx inngest-cli@latest dev -u http://127.0.0.1:5001/inngest-firebase-functions/us-central1/inngest
+```
+
+_The above command example features a project named `inngest-firebase-functions` deployed on the `us-central1` region_
```
Comment (sylwiavargas) [inline pages/docs/sdk/serve.mdx:442]:
```suggestion
_The above command example features a project named `inngest-firebase-functions` deployed on the `us-central1` region_.
```

**Review by sylwiavargas — APPROVED** [review]
Left one last comment with a missing "."
Otherwise, good to go 🥳

**Pull request merged**